domains/park.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "parknich",
+           "email": "",
+           "discord": "556850704770138114",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.RMlyfY4mGtUlaUQkOaxoaNdyfU8ldylf06Qtl7tYhrU_ivjmCSHks5pygP37oEYyHN6NfenY9FntFxaddwKCbb5eGeGs7cO4M213MPs9agSU-3NzvjLg6Kya5bjVrlhcF8KVXihW5ITNp4MUA20gPuM2EfImenDGnwe0azXhSuiFbaDvak1Yg51jy3eAnKi3OUgPF8TfOhHO_Hu4bGg5yq87P8Sp1X7w6ZhThgX83ZUCdH16xHIa_oUhnT7AQZIrZlLyT6SSDRg7oryZVPLlD1E-BrqEQlZXPuZPtDRxrNXmISCSorBCDd5fGAAEHN_RG_8qs2ysJoDTQ9E_g5v23g.8cbDPSl8X94DpagdJchXVw.rSdmAuGBzGk33r1XVbD6BkPI6CAi7BQ5lrXfTmuEHXvNELMytmHQL4uGzXgyKoAsQYbpRhwtoug6vE48VA2ydL08AgwlX3rdp9W_20E_gtgWZu36-2Diqy75WHbl567B.uQeFnbQ5lEE89z14li7Rig"
+        },
+    
+        "record": {
+            "CNAME": "https://bio.parknich.xyz/"
```

Comment:
CNAMEs cannot contain protocols (`https://`). 
please read our [documentation](https://is-a.dev/docs)